gmso/tests/base_test.py

```diff
@@ -97,8 +98,34 @@ def water_system(self):
 
         packed_system = mb.fill_box(
                 compound=water,
-                n_compounds=10,
+                n_compounds=2,
                 box=mb.Box([2, 2, 2])
                 )
 
         return  from_mbuild(packed_system)
+
+    @pytest.fixture
+    def typed_water_system(self, water_system):
+        top = water_system
+
+        ff = ForceField(get_path('tip3p.xml'))
+
+        element_map = {"O": "opls_111", "H": "opls_112"}
+
+        for atom in top.sites:
+            atom.atom_type = ff.atom_types[atom.name]
+
+        for bond in top.bonds:
+            bond.bond_type = ff.bond_types["opls_111~opls_112"]
```

Comment:
It doesn't look like `Bond` doesn't have a `bondtype` attribute. It only has `connection_type`. 
```suggestion
            bond.connection_type = ff.bond_types["opls_111~opls_112"]
```

https://github.com/mosdef-hub/gmso/blob/aa1d2a43719dae6e1c1a21ec5c9e2b4a1a44ffbf/gmso/core/bond.py#L33